test/file.jl

```diff
@@ -1156,3 +1156,70 @@ if !Sys.iswindows()
         test_22566()
     end
 end  # !Sys.iswindows
+
+function file_prefix_test(path, prefix, pfxlen=length(prefix), fails=false)
+    if Sys.iswindows() && (pfxlen > 3)
```

Comment:
Why not also set `pfxlen = 5` on Unix? Then you wouldn't need that argument at all.